### PR TITLE
Sprite and Loadout updates [RTM]

### PR DESCRIPTION
**Pull request by thevandie**
## About The Pull Request

Updates a load of sprites with both my own handiwork and stuff from the archive Valo has given me (some is Walter's, some is from other random people, I can't find em). I also modified most of the drink reagent colors to look less bad, because for some reason 99% of drinks used whiskey's color, and it just... looked like shit. Literal shit. SO, now they look closer to their irl counterparts or sprite

(I deleted the drink sprite changes, so now they just use their old sprites. I didn't change the reagent colors because when you see vodka in a beaker it shouldn't be blueberry puree blue, it should be clear with a slight blue tint (for creative license purposes)

## Why It's Good For The Game

All of these sprites were outdated and looked like shit compared to the new artstyle, so I have updated them. Especially the mountain wind and space-up full-sized bottles. Those were... Eugh.

## Testing

I deleted .dmb, reran build.bat, loaded up the game, and made sure the thing I changed worked everytime I modified something. It worked fine each time.

## Changelog
:cl: Vandu-jr, Cringelord, Watler'Je, de_dust_2
add: new sprites for the Generic Robot limbs
add: generic robot torso, groin, and head to the augmentation menu in setup
add: new sprites for the jar
add: new sprites for the Space Mountain Wind (Mountain Dew) and Space Up (7-Up) full-sized bottles
add: Space Mt. Wind, Space Cola, and Space-Up full-sized bottles to the soda vending machine for 100 credits (they contain 100 units each)
add: new sprites for roach and kaiser beer
add: new sprites for the cigarettes
add: new sprites for the generic, medical, and dinnerware vendors
add: new sprites for the Emeraldine and Miss Blue Caracao bottles
add: the security hud sunglasses to the loadout for IH
tweak: all of the common alcohols have had their colors remapped to something more fitting along with making them no longer use their unique sprites (because they use default one)
tweak: the jar now uses a different path so it functions properly like a beaker
fix: fixed a bug with the beaker's reagent-filling sprites, it stuck out a pixel below the actual beaker when completely full
/:cl:

image
image
image
image
image
image
image
image

### CODE REVIEW

**Comment by thevandie** [comment]
I love stealing sprites and adding them into the game

**Comment by Firefox13** [comment]
> I love stealing sprites and adding them into the game

Credit sprite authors properly, add their names to the changelog.

**Comment by thevandie** [comment]
I did
I have zero idea who made the other ones, and there's no info for me to find out

**Comment by Firefox13** [comment]
> archive Valo has given me (some is Walter's, some is from other random people, I can't find em)

> 🆑 Vandu-jr, Cringelord, Valo's Archive (not valo, just the archive he gave me)

No, you did not.

I still don't like beer cup. If you intend on resorting all glass sprites it's better to do it in a separate PR for all glasses instead of leaving it as a white crow.
Bottles can stay though.

**Comment by Firefox13** [comment]
You can also, like, ask Valo.

**Comment by thevandie** [comment]
> I still don't like beer cup.

90% of other people do, but alright...

**Comment by thevandie** [comment]
I fixed all your problems with it

**Comment by Firefox13** [comment]
> tweak: all of the common alcohols have had their colors remapped to something more fitting along with making them no longer use their unique sprites (I explained why earlier)

Care to explain again? Is this because they lack unique sprite so they use default glass_brown (for failed drinks)?

**Review by Firefox13 — APPROVED** [review]
Seems to be the case. Code and sprites look fine, I edited the changelog for readability - please remember that it needs to be on-point as it goes into the game (for younger generations to read).

**Comment by thevandie** [comment]
> Care to explain again? Is this because they lack unique sprite so they use default glass_brown (for failed drinks)?

No, it's because their reagent color is ass
Like when you put it in a beaker

**Comment by Firefox13** [comment]
yeah.

**Comment by thevandie** [comment]
> Seems to be the case. Code and sprites look fine, I edited the changelog for readability - please remember that it needs to be on-point as it goes into the game (for younger generations to read).

Ok thanks

File: code/modules/reagents/reagent_containers/food/drinks/jar.dm [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
-		desc = "A jar. You're not sure what it's supposed to hold."
+// /obj/item/reagent_containers/food/drinks/jar/on_reagent_change()
+//	if (reagents.reagent_list.len > 0)
+//		icon_state ="jar_what"
+//		name = "jar of something"
+//		desc = "You can't really tell what this is."
+//	else
+//		icon_state = "jar"
+//		name = "empty jar"
+//		desc = "A jar. You're not sure what it's supposed to hold."
```
Comment (hyperioo) [inline code/modules/reagents/reagent_containers/food/drinks/jar.dm:21]:
```suggestion
```
No point in keeping that laying around.

**Review by hyperioo — APPROVED** [review]

File: code/modules/reagents/reagent_containers/food/drinks/jar.dm (hunk 1 above)
Comment (thevandie) [reply code/modules/reagents/reagent_containers/food/drinks/jar.dm]: Fair enough, gimme a second

**Pull request merged**